app/com/linkedin/drelephant/tuning/AbstractAlgorithmManager.java

```diff
@@ -0,0 +1,44 @@
+package com.linkedin.drelephant.tuning;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+
+
+public abstract class AbstractAlgorithmManager implements Manager {
```

Comment:
Algorithm=>Tuning/TuningType? We are using the term tuning type to differentiate between HBT/OBT. OBT will then have further algorithms. Keep the code consistent with that terminology?

Reply:
agreed .